### PR TITLE
add terraform demo

### PR DESCRIPTION
**Pull request by fuku2014**

### CODE REVIEW

File: terraform/demo/.gitignore
Comment (kozm) [inline terraform/demo/.gitignore:1]: i think so this file is not appropriate for sample code.
coudle you delete this file?

File: terraform/demo/files/publickey.pem
Comment (kozm) [inline terraform/demo/files/publickey.pem:1]: i think so this file is sensitive file.

could  you replace to dummy file.

File: terraform/demo/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Terraform Demo
+
+This is a simple terraform demo for [NIFCLOUD](https://www.nifcloud.com/)  
+This repository uses [Terraform Provider for NIFCLOUD](https://github.com/nifcloud/terraform-provider-nifcloud)
+
+## Usage
+
+To use this example, you need to add environment variables  with the following content:
```
Comment (kozm) [inline terraform/demo/README.md:8]: i'd like to not include sensitive file this project so please add publickey create step

File: terraform/demo/.gitignore
Comment (fuku2014) [reply terraform/demo/.gitignore:1]: Thanks!

FIx it!

File: terraform/demo/files/publickey.pem
Comment (fuku2014) [reply terraform/demo/files/publickey.pem:1]: Thanks!

FIx it!

**Comment by kozm** [comment]
thanks for fix

File: terraform/demo/README.md (hunk 1 above)
Comment (fuku2014) [reply terraform/demo/README.md:8]: using tls_private_key module!

**Review by kozm — APPROVED** [review]

**Pull request merged**